modules/cij/extractors/__init__.py

```diff
@@ -0,0 +1 @@
+from cij.extractors import fio_json_read
```

Comment:
Why is this needed?

Reply:
Hmm, that's a good question. I'm using it to make `fio_json_read` available as an import from `cij.extractors`. It's quite possible that we can remove this, but then I think we'll have to change the way that imports of the extractors work.

It would be quite nice to not have to add an import for each extractor we add - it's very easy to forget

Reply:
Yeah, the not so pretty thing to do would be in ``__init__`` doing:
``from cij.extractors import *``

Some PEP will probably complain about that :)

Reply:
This should be fixed now - changed the way I'm listing extractors